tests/conftest.py

```diff
@@ -159,25 +156,22 @@ def __eq__(self, other):
         b = _strip_and_dedent(other)
         if a == b:
             return True
-        else:
-            self.explanation = _make_explanation(a.splitlines(), b.splitlines())
-            return False
+        self.explanation = _make_explanation(a.splitlines(), b.splitlines())
+        return False
 
 
 def _sanitize_general(s):
     s = s.strip()
     s = s.replace("pybind11_tests.", "m.")
-    s = _long_marker.sub(r"\1", s)
-    return s
+    return _long_marker.sub(r"\1", s)
 
 
 def _sanitize_docstring(thing):
     s = thing.__doc__
-    s = _sanitize_general(s)
-    return s
+    return _sanitize_general(s)
```

Comment:
Overwriting a variable is best to avoid if possible. The new form is nicer because `s` is now longer being reassigned. (The only problem SIM was reporting was that a variable was being assigned then immediately returned. But avoiding a reassignment is a big plus IMO.)

Reply:
> But avoiding a reassignment is a big plus IMO.

Agreed.